Sources/MapboxNavigation/NavigationViewControllerDelegate.swift

```diff
@@ -11,7 +11,7 @@ import Turf
  
  For convenience, several location-related methods in the `NavigationServiceDelegate` protocol have corresponding methods in this protocol.
  */
-public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
+public protocol NavigationViewControllerDelegate: VisualInstructionDelegate, NavigationTilesVersioningDelegate {
```

Comment:
Is it a good idea to provide a common point with `NavigationTilesVersioningDelegate` only for highest level delegates/objects? And for `PassiveLocationManager` you've added a delegate property, but for `NavigationViewController` it's done by extending an existing delegate.

Reply:
Good question. I thought that `NavigationViewControllerDelegate` covers various aspects of the Navigation process, so  handling switching may fall there too. And `PassiveLocationManager.delegate` is actually a `LocationProviderDelegate` which is designed for a specific purpose and tiles logic does not fit to that context.